scottofthemonth.html

```diff
@@ -263,29 +263,172 @@
           transform: scale(1.5);
         }
       }
-    </style>
```

Comment:
It would be nice to extract this into an unused (as of now) js file that we could import here (rather than deleting it - since this is Sweden, and who knows when it will snow next).